hana/package.json

```diff
@@ -20,7 +20,8 @@
     "npm": ">=8"
   },
   "scripts": {
-    "test": "npm start && jest --silent",
+    "test": "npm start && npm run jest",
+    "jest": "jest --silent",
```

Comment:
This desperation is causing trouble in the pipeline as it is not able to forward the `--maxWorkers=1` argument which prevents the database isolation race condition.

Reply:
I was only trying to find a solution to omit the start script for local runs :)